llvm/test/Transforms/LoopVectorize/unused-interleave-group.ll

```diff
@@ -0,0 +1,45 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 3
+; RUN: opt -passes=loop-vectorize -S -force-vector-width=4 -enable-interleaved-mem-accesses=true -debug-only=loop-vectorize < %s 2>&1 | FileCheck %s
+
```

Comment:
This needs `; REQUIRES: asserts` so the test only runs when assertions are enabled, which makes sure `-debug` is available (in regular release builds, `-debug` isn't available)

Reply:
Fixed. Thanks for the reminding!